src/index.ts

```diff
@@ -35,12 +51,29 @@ function showPrivacyManager() {
 	initialised.then(() => CMP?.showPrivacyManager());
 }
 
-export const cmp = {
-	init,
-	willShowPrivacyMessage,
-	showPrivacyManager,
+// *************** START commercial.dcr.js hotfix ***************
+// export const cmp = {
+// 	init,
+// 	willShowPrivacyMessage,
+// 	showPrivacyManager,
+// };
+
+// export { onConsentChange } from './onConsentChange';
+
+const actualExports = {
+	cmp: { init, willShowPrivacyMessage, showPrivacyManager },
+	onConsentChange: actualOnConsentChange,
 };
 
-export { onConsentChange } from './onConsentChange';
+if (window) {
```

Comment:
Shouldn't that be part of the init function to avoid assigning `guCmpHotFix` more than once?

Reply:
There's a `window.guCmpHotFix.initialised` value set on L28 to check it only gets initialised once.

Reply:
I'm not sure that protects against double initialisation in the case the code is loaded as part of two different JS 
 bundles, as this window assignment is globally scoped?

Reply:
You're right, and that's why we spread using the spread operator below: `...window.guCmpHotFix`